data/json/items/ammo/22.json

```diff
@@ -53,13 +53,18 @@
   },
   {
     "id": "22_ratshot",
-    "copy-from": "22_cb",
+    "copy-from": "22_lr",
     "type": "AMMO",
     "name": ".22 rat-shot",
     "name_plural": ".22 rat-shot",
     "description": "A .22 caliber cartridge loaded with very small pieces of shot contained within a fragmenting plastic capsule.  It has an extremely short range and penetration ability, but it can hit targets with greater ease.",
-    "range": 3,
-    "proportional": { "recoil": 0.5 },
+    "proportional": {
+      "count": 0.20,
+      "range": 0.6,
+      "damage": 0.5,
+      "dispersion": 1.3,
```

Comment:
copy-paste error. It turns out that I cannot test these changes via mod, as multiple copy-froms seem to stop working after a few steps.